test/src/tests/mocks/BasketMgr.js

```diff
@@ -0,0 +1,222 @@
+                                masterProduct: this.product.getAllCategories(),
+                                getAdjustedMerchandizeTotalPrice: this.getAdjustedMerchandizeTotalPrice(),
+                                optionProductLineItems: [
+                                    {
+                                        optionID: 'optionId1',
+                                        optionValueID: 'selectedValueId1',
+                                        productName: 'productName1',
+                                        lineItemText: 'lineItemText1',
+                                        basePrice: {
+                                            value: .99
+                                        }
+                                    },
+                                    {
+                                        optionID: 'optionId2',
+                                        optionValueID: 'selectedValueId2',
+                                        productName: 'productName2',
+                                        lineItemText: 'lineItemText2',
+                                        basePrice: {
+                                            value: 1.99
+                                        }
+                                    }
+                                ],
+                                bonusProductLineItem: {
+                                    originalPrice: 100,
+                                    price: 50,
+                                    originalPriceValue: 100,
+                                    priceValue: 50
+                                },
+                                bundledProductLineItems: [
+                                    {
+                                        isProdBundle: true,
+                                        prodBundleIDs: [
+                                            "sony-ps3-consoleM",
+                                            "easports-nascar-09-ps3M",
+                                            "easports-monopoly-ps3M",
+                                            "namco-eternal-sonata-ps3M",
+                                            "sony-warhawk-ps3M"
+                                        ]
+                                    }
+                                ]
+                            },
+                        ]
+                    },
+                }
+            },
+
+            getPaymentInstruments: () => {
+                const paymentInstruments = [{
+                    creditCardNumber: '411111111111',
+                    creditCardType: 'visa',
+                    creditCardExpirationMonth: '03',
+                    creditCardExpirationYear: '30',
+                    creditCardNumberLastDigits: '1111',
+                    maskedCreditCardNumber: '############',
+                }]
+                return paymentInstruments
+            },
+
+            // getShippingLineItems: () => {
```

Comment:
Are these commented out lines still needed?

Reply:
@cykolln same here, I will update and create a PR.